src/components/CommandBar/CommandBar.tsx

```diff
@@ -94,9 +102,6 @@ export const CommandBar = () => {
         commandBarActor.send({ type: 'Deselect command' })
       }
     } else {
-      const entries = Object.entries(selectedCommand?.args || {}).filter(
-        (a) => !a[1].hidden
-      )
```

Comment:
I have a vague sense that this is used to ensure the `nodeToEdit` arg is never seen, but I just need to user test it in a bit

Reply:
nodeToEdit should always be hidden so it still should be part of the filtered out items there